content/html/glossaries/html-tag-article.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: "<article>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Semantic"
+  - "HTML Tags"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
+---
+
+## Definition 
+
+The semantic HTML tag `<article>` represents a part of a page which is self-contained, and could be published elsewhere. Some common uses of the `<article>` tag  include blog posts or magazine articles.
```

Comment:
The `<article>` semantic tag represents a part of a page that is self-contained and could be published elsewhere. Some common uses include blog posts or magazine articles.